_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -816,11 +816,13 @@ Organizing portfolios around development value streams enables visualizing the f
 **IT Solution**
 
 An IT solution is a combination of one or more IT Products.
-It produces the environment within which end-users operates.
+It produces the environment within which end-users operates.  
 
-![How IT Solutions, IT Products, software, and infrastructure relate to each other]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
+Figure 1. Relationship Model between the different software elements (part 1).  
 
-**IT Product**
+![Below is a flowchart with five elements connected to each other.  The flowchart has the following elements displayed in the box: IT Solution, IT Component, Software, Infrastructure. The flow goes from left to right and connects those elements with an arrow. The Configuration element is located below the Software and Infrastructure elements, the latter are pointing to it from both sides. ]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
+
```

Comment:
This image also appears in ABB and @gcharest  just added alt text for it as follows. Recommend consistency and using the same alt text. 
"An UML diagram representing the hierarchical relation of an IT Solution and its components.
There are 5 rectangles on the diagram connected by relationship lines.
The first rectangle to the left is titled "IT Solution".
It is connected to a second rectangle, titled "IT Product", located on its right by a relationship line; the end of the line starting from the "IT Solution" rectangle has a "1" and the other end has "1...*" and a small white diamond.
The "IT Product" rectangle is connected to a third rectangle, titled "Software", located on its right by a relationship line; the end of the line starting from the "IT Product" rectangle has a "1" and the other end has a "1...*" and a small white diamond.
The "Software" rectangle has two relationships connections.
The first connection is to a rectangle, titled "Configuration", located below the "Software" rectangle; the end of the line starting from the "Software" rectangle has a "1" and the other end has a "1...*" and a small black diamond.
The second connection is to another rectangle, titled "Infrastructure", located to the right of the "Software" rectangle; the end of the line starting from the "Software" rectangle has a "1" and the other end has a "1...*" and a small white diamond.
There is one last connection, linking the "Infrastructure" and "Configuration" rectangles; the end of the line starting from the "Infrastructure" rectangle has a "1" and the other end has a "1...*" and a small black diamond."

Reply:
nice catch!